src/configurations/destinations/comscore/db-config.json

```diff
@@ -12,41 +12,59 @@
       "blacklistedEvents",
       "whitelistedEvents",
       "eventFilteringOption",
+      "fieldMapping",
       "oneTrustCookieCategories",
       "consentManagement",
       "ketchConsentPurposes"
     ],
     "excludeKeys": [],
-    "supportedSourceTypes": ["android", "ios"],
+    "supportedSourceTypes": ["android", "ios", "web"],
     "supportedMessageTypes": {
       "device": {
         "android": ["identify", "track", "screen"],
-        "ios": ["identify", "track", "screen"]
+        "ios": ["identify", "track", "screen"],
+        "web": ["page"]
       }
     },
     "supportedConnectionModes": {
       "android": ["device"],
-      "ios": ["device"]
+      "ios": ["device"],
+      "web": ["device"]
     },
     "destConfig": {
       "defaultConfig": [
-        "publisherId",
+        "publisherId"
+      ],
+      "android": [
         "appName",
         "foregroundAndBackground",
         "autoUpdateInterval",
         "foregroundOnly",
         "blacklistedEvents",
         "whitelistedEvents",
```

Comment:
These three fields are source(iOS, Android, JS) agnostic, please keep them in the default config.

Reply:
Revert your changes in the schema.json for the same